packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -341,6 +342,48 @@ const toEqualFile = () => ({
   },
 });
 
+const syncTestBodyBuilder = task => () => {
+  function runTask() {
+    run(task);
+  }
+
+  const throwMsg = task.options.throws;
+  if (throwMsg) {
+    // internal api doesn't have this option but it's best not to pollute
+    // the options object with useless options
+    delete task.options.throws;
+
+    assert.throws(runTask, function (err) {
+      assert.ok(err.message.indexOf(throwMsg) >= 0);
```

Comment:
```
return err.message.indexOf(throwMsg) >= 0;
```
is also changed to `assert.ok`. See https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message


>  Avoid returning anything from validation functions besides `true`.
    Otherwise, it's not clear what part of the validation failed. Instead, throw an error about the specific validation that failed (as done in this example) and add as much helpful debugging information to that error as possible.